### ISSUE TITLE
SWIFT Catalogue web interface - reduce whitespace in NWCSAF generated images

### ISSUE THREAD

**Issue by tdjames1** [cemac/SWIFT-Testbed3#20]
Some NWCSAF images are plotted with a lot of whitespace around the plot. We want to automatically crop the images and resize so that the legend is in proportion with the plot.

![WP4_NWCSAF](https://user-images.githubusercontent.com/10053102/108056283-f7a9f900-7048-11eb-88c4-1a572a82996f.png)

**Comment by earajr** [comment]
I have been testing out how to shrink the colourbars in NWCSAF plots and for most plots I have managed to do this satisfactorily to reduce the whitespace. This only required some fairly minor edits to one of the plotting routines but has taken some time testing to make sure the results are acceptable.

**Comment by tdjames1** [comment]
Sounds good. Where are you working on this - is there a github repo for the NWCSAF plotting code?

**Comment by earajr** [comment]
No, there probably should be. I was just playing around using a copied directory on tegan.

**Comment by lecjlg** [comment]
Might be an idea as i am currently implementing the plotting on Jasmin for the redundant system, it would make sense to pull these from git to keep them the same

**Comment by tdjames1** [comment]
@earajr Could you let me have some samples of the new layouts for use in the web UI mockup?

**Comment by earajr** [comment]
Hi Tamora, I actually updated the scripts on the Chilbolton machine too. The latest images in the catalogue are with the white space significantly reduced.
Heres the page for West Africa today https://sci.ncas.ac.uk/swift/categories/view/500/2021-03-03

**Comment by tdjames1** [comment]
Thanks, Alex!

**Comment by tdjames1** [comment]
Do you think it would be possible to standardise the sizing of the images? At the moment there seem to different sizes for different products e.g. CRR 619 x 584px vs RDT 769 x 680px. Resizing these to a standard width would be helpful for the web UI and if this was done initially I think it would produce better and more consistent images than if images are resized during post-processing.
